src/ldc/simd.di

```diff
@@ -204,14 +209,44 @@ if(is(typeof(llvmVecType!V)))
     alias BaseType!V T;
     enum llvmT = llvmType!T;
     enum llvmV = llvmVecType!V;
-    enum ir = `
-        %p = bitcast `~llvmT~`* %0 to `~llvmV~`*
-        %r = load `~llvmV~`* %p, align 1
-        ret `~llvmV~` %r`;
+    version (LDC_LLVM_PRE307)
+        enum ir = `
+            %p = bitcast `~llvmT~`* %0 to `~llvmV~`*
+            %r = load `~llvmV~`* %p, align 1
+            ret `~llvmV~` %r`;
+    else
+        enum ir = `
+            %p = bitcast `~llvmT~`* %0 to `~llvmV~`*
+            %r = load `~llvmV~`, `~llvmV~`* %p, align 1
+            ret `~llvmV~` %r`;
 
     alias inlineIR!(ir, V, T*) loadUnaligned;
 }
 
+unittest
```

Comment:
unittesting is not done for *.di files. So we either have to enable that in the cmake script or move these tests elsewhere. In my opinion, it is much nicer to remove this testing "clutter" out of this file and into an IR test (possibly with an ASM test).